src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -33,6 +33,16 @@
 
 namespace Dynatrace.OpenTelemetry.Exporter.Metrics.Tests
 {
+	public sealed class HistogramTestData
```

Comment:
Nit: We could place this inside the `MinMaxEstimationData` class, to be close to where it's instantiated. Also fine as is, your call :)

Reply:
Done in [0d7b371](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/32/commits/0d7b371c7664476ac2c55fe0ff29c1465684e471). :slightly_smiling_face: